tests/integration/test_ethereum_tester.py

```diff
@@ -190,6 +190,8 @@ def test_eth_getTransactionReceipt_unmined(self, eth_tester, web3, unlocked_acco
             eth_tester.enable_auto_mine_transactions()
             eth_tester.mine_block()
 
+    # TODO: replace auto mining switch with decorator
+    # TODO: replicate for goethereum tests
```

Comment:
*if* it turns out that testing this in the integration tests is messy lets look at other options and whether it's even necessary before you burn too much time trying to make it work.  Since this functionality will likely behave differently on different clients, and it's possible that some clients won't reliably accept the replacement transaction I could see these tests getting really messy.

Reply:
It seems like the `--fakepow` option to geth is what is mining the blocks for the integration tests? There isn't a way to disable that through a RPC or console call so if we wanted to do it, i believe we would need to recreate the geth process for those test specifically. The other option would be to use the miner and call `miner.start()` and `miner.stop()` similar to the `eth-tester` tests. Not sure how this would affect other tests though. Just an initial look, i will need to have a closer look at this.

I guess if the API promises to support a client then we would need to know if the integration breaks so I can't really say that it's unnecessary.

Reply:
We had major issues with tests timing out when we tried to do `miner.start/stop()` in our CI environment.  I'd rather skip testing this stuff in integration tests since the tests become unreliable if we're required to mine.

Reply:
I've made these tests pass for the geth integration tests :)